sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -228,6 +228,16 @@ def test_windowed_batches(self):
               7,  # elements in [30, 47)
           ]))
 
+  def test_sized_batches(self):
```

Comment:
I think when someone looks at this in the future and says, "why is this here", you could illustrate it better with a test like this (I'm not sure if this works but you get the point).

def test_element_size_fn()
""" Tests passing different sized arrays as elements and then weighting them by how large the arrays are. Batch sizes will depend on the size of the arrays within."""
...
| beam.Create([[1]*1, [1] *1, [10] *10, [6] * 6 ...], reshuffle=False)
| util.BatchElements(min_batch_size =10, max_batch_size=10, element_size_fn=lambda x: len(x))
| beam.Map(len)
assert_that(res, equal_to([12, 11, 10]))
...

I think that might just make it clearer to the future reader as to why this feature might exist.  Feel free to take or ignore this suggestion.

Reply:
Good idea. The mix of `[]`'s and `*`'s are a bit hard for me to read, so I'm using strings.